db/migrate/20180502153341_map_gds_content_id_from_govuk.rb

```diff
@@ -0,0 +1,7 @@
+class MapGdsContentIdFromGovuk < ActiveRecord::Migration[5.1]
+  def change
+    org = Organisation.find_by(name: 'government-digital-service')
+    return unless org
+    org.update_attribute(:govuk_content_id, 'af07d5a5-df63-4ddc-9383-6a666845ebe9')
```

Comment:
Do we need content ids for the other organisations mentioned in the acceptance criteria of https://trello.com/c/Jy6YRdzO/157-make-orgs-work ?

Reply:
We do indeed - I'm going to put them in the next PR. The integration IDs should match the staging/production ones.